solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureMetadata.java

```diff
@@ -58,14 +59,20 @@ public void getInstanceTest() throws FeatureException
         .getClass().getCanonicalName());
   }
 
-  @Test(expected = FeatureException.class)
-  public void getInvalidInstanceTest() throws FeatureException
+  @Test
+  public void getInvalidInstanceTest()
   {
-    final Map<String,Object> map = new HashMap<String,Object>();
-    map.put(Feature.NAME_KEY, "test");
-    map.put(Feature.CLASS_KEY, "org.apache.solr.ltr.feature.LOLFeature");
-    store.addFeature(map, "testFstore2");
-
+    final FeatureException expectedException = 
+        new FeatureException("Error loading class 'org.apache.solr.ltr.feature.LOLFeature'");
```

Comment:
Underneath our error message should probably say "Error creating <FEATURE_NAME> could not load class ..." This would just make our lives easier.